VCFv4.3.tex

```diff
@@ -1380,6 +1380,19 @@ \subsection{Representing unspecified alleles and REF-only blocks (gVCF)}
 \end{flushleft}
 \normalsize
 
+\subsection{Describing family relationships}
```

Comment:
I would prefer to make this part of the PEDIGREE meta entry description. I can also address the TODO.

Reply:
Pedigree lines are putatively used in two separate scenarios: clonal relationships and trios/families. There is already a separate section covering the clonal scenario, so it makes some sense to cover the other one in its own separate section too.

Similarly the minimal §1.4.8 META and SAMPLE meta entry description is expanded upon by §5.4.10.

Reply:
The only issue I see is that the clonal relationships are described in the context of breakends and are subsections of that one. On the other hand, this would be a completely separate one which would perfectly fit in the PEDIGREE section.